guides/testing/supertest.md

```diff
@@ -0,0 +1,7 @@
+# Supertest
+
```

Comment:
probably good to explain that we use supertest for e2e testing :)

Reply:
Ah yeah that's true, let me add that

Reply:
Added @godwinpang